pcsx2/gui/MainFrame.cpp

```diff
@@ -491,12 +488,13 @@ void MainEmuFrame::CreateConfigMenu()
 
 	m_menuConfig.Append(MenuId_ChangeLang, L"Change &Language..."); // Always in English
 	m_menuConfig.Append(MenuId_Config_ResetAll, _("C&lear All Settings..."),
-						AddAppName(_("Clears all %s settings and re-runs the startup wizard.")));
+		AddAppName(_("Clears all %s settings and re-runs the startup wizard.")));
 }
 
 void MainEmuFrame::CreateWindowsMenu()
 {
 	m_menuWindow.Append(MenuId_Debug_Open, _("&Show Debug"), wxEmptyString, wxITEM_CHECK);
+	m_menuWindow.Append(MenuId_Debug_CreateBlockdump, _("Create &Blockdump"), _("Creates a block dump for debugging purposes."), wxITEM_CHECK);
```

Comment:
Move the blockdump on top and maybe rename Show Debug to Show Debugger.

Reply:
sure :+1: